packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-before-connect.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-warning-comments */
+
+import '@ciscospark/plugin-phone';
+
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import setValue from '../lib/set-value';
+
+describe(`sample-browser-multi-party-call`, () => {
+  describe(`dial before connecting`, () => {
+    let mccoy, roomId, spock;
+
+    const browserSpock = browser.select(`browserSpock`);
+    const browserMccoy = browser.select(`browserMccoy`);
+
+    const users = {
+      mccoy: null,
+      spock: null
+    };
+
+    before(`create users and register`, () => testUsers.create({count: Object.keys(users).length})
+      .then((created) => Promise.all(Object.keys(users).map((name, index) => {
+        const user = users[name] = created[index];
+        user.spark = new CiscoSpark({
+          credentials: {
+            authorization: user.token
+          }
+        });
+
+        console.log(user.spark);
+
+        return user.spark.phone.register();
+      })))
+      .then(() => {
+        mccoy = users.mccoy;
+        spock = users.spock;
+      }));
+
+    beforeEach(`enable group calling`, () => {
+      spock.spark.config.phone.enableExperimentalGroupCallingSupport = true;
+      mccoy.spark.config.phone.enableExperimentalGroupCallingSupport = true;
+    });
+
+    afterEach(`disable group calling`, () => {
+      spock.spark.config.phone.enableExperimentalGroupCallingSupport = false;
+      mccoy.spark.config.phone.enableExperimentalGroupCallingSupport = false;
+    });
+
+    step(`create test room and add members to room`, () => {
+      spock.spark.request({
+        method: `POST`,
+        service: `hydra`,
+        resource: `rooms`,
+        body: {
+          title: `Call Test`
+        }
+      })
+        .then((res) => {
+          const room = res.body;
+
+          roomId = room.id;
+
+          return spock.spark.request({
+            method: `POST`,
+            service: `hydra`,
+            resource: `memberships`,
+            body: {
+              roomId: room.id,
+              personId: mccoy.id
+            }
+          });
+        });
+    });
+
+    step(`loads the app`, () => {
+      browser.reload();
```

Comment:
don't use reload, it's pretty slow because it establishes a completely new selenium session. Adam indicated there's also a `browser.refresh()`